scripts/update-rules.js

```diff
@@ -15,6 +25,29 @@ const tablePlaceholder = /<!--RULES_TABLE_START-->[\S\s]*<!--RULES_TABLE_END-->/
 const EMOJI_STAR = ':white_check_mark:';
 const EMOJI_OCTANE = ':car:';
 const EMOJI_STYLISTIC = ':dress:';
+const EMOJI_FIXABLE = ':wrench:';
+
+function isRuleFixable(ruleName) {
+  const relativePath = ruleName.startsWith('deprecated-')
+    ? `../lib/rules/deprecations/${ruleName}.js`
+    : `../lib/rules/${ruleName}.js`;
+  const pathRule = path.resolve(__dirname, relativePath);
+  let rule = fs.readFileSync(pathRule, { encoding: 'utf8' });
+
+  let ast = parse(rule, { sourceType: 'module' });
+
+  let isFixable = false;
+
+  traverse(ast, {
+    Identifier(path) {
+      if (path.node.name === 'isFixable') {
```

Comment:
The rule `require-form-method` specifies `isFixable: false` (and the rule is not fixable) so we need to ignore if the value is `false`.

In the future, it would be great if we added a static meta config to rules like eslint rules have, indicating if they are fixable, so we could avoid trying to parse it.